### DISCUSSION TITLE
Old state file version

### DISCUSSION THREAD

**Discussion by tbojustin** [GoogleCloudPlatform/terraformer#1128] [Q&A] [answered]
Hey, I'm new to all of this and some friends pointed out this tool. I've started working with what it generated, but it seems to create for an older version of Terraform (0.12) . Am I doing something wrong?

**Comment by josborne** [comment, marked as answer]
I noticed this too.  I had to download 0.13.7 from https://releases.hashicorp.com/terraform/ to work with the output from terraformer correctly.

**Reply by tbojustin** [reply]
Thanks @josborne. I didn't have the level of dedication, because the project I had was newish. I've started a blog to explain what I hacked together to get this to work in the latest version.

This tool was really useful, actually, Getting to the end just took a little more elbow grease that I'd expected.
